libs/novui/src/tokens/semanticSpacing.tokens.ts

```diff
@@ -15,4 +15,282 @@ export const SEMANTIC_SPACING_TOKENS = defineSemanticTokens.spacing({
       },
     },
   },
+  paddings: {
+    page: {
+      horizontal: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+      vertical: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+    },
+    panel: {
+      horizontal: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+      vertical: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+    },
+    popover: {
+      horizontal: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+      vertical: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+    },
+    node: {
+      horizontal: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+      vertical: {
+        type: 'spacing',
+        value: '{spacing.150}',
+      },
+    },
+    nav: {
+      menu: {
+        horizontal: {
+          type: 'spacing',
+          value: '{spacing.100}',
+        },
+        vertical: {
+          type: 'spacing',
+          value: '{spacing.100}',
+        },
+      },
+      header: {
+        horizontal: {
+          type: 'spacing',
+          value: '{spacing.50}',
+        },
+        vertical: {
+          type: 'spacing',
+          value: '{spacing.50}',
+        },
+      },
+      futer: {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏Is this a legitimate semantic name, or a typo of `footer`?

Reply:
@rifont I believe it is a typo -- but it's how it's currently done in Figma. I'm not happy with a number of the tokens and how they're written, but I elected to keep everything to match Nik's designs for now...

Reply:
Actually -- decided to fix this because it's a mistake -- not a convention